### PR TITLE
Import of MobileCoreServices improved for usage with CocoaPods. Mentioned in #46

### PR DESCRIPTION
**Pull request by lennartstolz**
As mentioned in #46  inside the  [Podspec Syntax Reference](http://guides.cocoapods.org/syntax/podspec.html#prefix_header_file) it's not recommended to use imports from the **LoopBack-Prefix.pch**.

> This attribute is not recommended as Pods should not pollute the prefix header of other libraries or of the user project.

Because **MobileCoreServices** are only used in **LBFile.m** I added the import there.

Best Lennart

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by hideya** [comment]
Thank you, @lennartstolz , for doing this!  The changes look good to me.

@raymondfeng , would you please review this?

File: LoopBack/LoopBack-Prefix.pch [hunk 1]
```diff
@@ -3,6 +3,5 @@
 //
 
 #ifdef __OBJC__
-    #import <MobileCoreServices/MobileCoreServices.h>
```
Comment (hideya) [inline LoopBack/LoopBack-Prefix.pch]: Would you please revert this change?
This prevents AFNetworking from performing "Automatic MIME type detection when uploading files".
https://github.com/strongloop/loopback-sdk-ios/blob/master/SLAFNetworking/SLAFHTTPClient.h#L89
You'll see the message as a warning when compiling the workspace.

**Comment by lennartstolz** [comment]
I'll revert the changes in pch- file and create a new pull request.

**Comment by hideya** [comment]
Thank you, @lennartstolz !
